src/main/java/com/formulasearchengine/mathmltools/mml/CMMLInfo.java

```diff
@@ -356,20 +305,18 @@ public final NodeList getChildNodes() {
         return cmmlDoc.getChildNodes();
     }
 
-    public final Double getCoverage(Multiset queryTokens) {
+    public final Double getCoverage(Multiset<String> queryTokens) {
         if (queryTokens.isEmpty()) {
             return 1.0;
         }
-        final Multiset<String> our = getElements();
+        final Multiset<String> our = getElements(true);
         if (our.contains(queryTokens)) {
             return 1.0;
         } else {
             final HashMultiset<String> tmp = HashMultiset.create();
             tmp.addAll(queryTokens);
             tmp.removeAll(our);
-            return 1
-                    - Double.valueOf(tmp.size())
-                    / Double.valueOf(queryTokens.size());
+            return 1 - (double) tmp.size() / (double) queryTokens.size();
```

Comment:
I have no direct source, but this should be a basic principle: inner functions first. This cast is a Widening Primitive Conversion and therefore called before the division.